pulpanalytics/management/commands/summarize.py

```diff
@@ -113,7 +114,9 @@ def handle(self, *args, **options):
                 .filter(created__gte=summary_start_datetime)
                 .filter(created__lt=summary_end_datetime)
             )
-            persistent_systems = systems.filter(age__gte=timedelta(days=1))
+            persistent_systems = systems.filter(
+                age__gte=timedelta(days=settings.PERSISTENT_MIN_AGE)
```

Comment:
```suggestion
                age__gte=timedelta(days=settings.PERSISTENT_MIN_AGE_DAYS)
```

I think units would be helpful here please.